pkg/osbuild/oscap_autotailor_stage.go

```diff
@@ -50,6 +50,12 @@ func NewOscapAutotailorStageOptions(options *oscap.TailoringConfig) *OscapAutota
 		return nil
 	}
 
+	// TODO: don't panic! unfortunately this would involve quite
+	// a big refactor and we still need to be a bit defensive here
+	if options.RemediationConfig.TailoringPath == "" {
```

Comment:
This might be a bit controversial. I've added it to its own commit for now, but happy to delete or squash depending on how we feel about it.